tests/Aspire.Hosting.Tests/Nats/AddNatsTests.cs

```diff
@@ -44,8 +44,10 @@ public void AddNatsContainerWithDefaultsAddsAnnotationMetadata()
     [Fact]
     public void AddNatsContainerAddsAnnotationMetadata()
     {
+        var path = OperatingSystem.IsWindows() ? @"C:\tmp\dev-data" : "/tmp/dev-data";
```

Comment:
Probably shouldn't hard code these paths.

Reply:
Why not?